agent/engine/docker_task_engine.go

```diff
@@ -758,3 +786,25 @@ func (engine *DockerTaskEngine) Capabilities() []string {
 func (engine *DockerTaskEngine) Version() (string, error) {
 	return engine.client.Version()
 }
+
+// enableConcurrentPull checks the docker version and return true if docker version > 1.9.0
+func (engine *DockerTaskEngine) enableConcurrentPull() bool {
+	version, err := engine.Version()
+	if err != nil {
+		seelog.Warnf("Failed to get docker version, err %v", err)
+		return false
+	}
+
+	match, err := utils.Version(version).Matches(">=1.9.0")
```

Comment:
There may be other issues with concurrent pulls than just that one.  I think we need to test each version that we plan to enable, and probably with more than one storage driver (aufs, devicemapper, and overlay would be a good selection).